docs/guides/src/operation/black_list.md

```diff
@@ -0,0 +1,61 @@
+# Black list
```

Comment:
`blacklist` is a terminology. So you'd better not separate it into two words.

Reply:
fixed